packages/block-library/src/navigation/editor.scss

```diff
@@ -288,6 +288,17 @@ $color-control-label-height: 20px;
 	}
 }
 
+// Selected "Start empty" state.
+// In this state, the block is entirely empty and will collapse to zero height unless there's
+// content inside. The below rule lets the placeholder stay in an invisible state to avoid
+// collapsing and shifting the layout.
+// @todo: This fix could be augmented with showing the generic appender when there are no
```

Comment:
I mentioned this in my comment below, but all this suggests to me we might want to shelve this PR and look at something like #38337 instead.

Reply:
I wouldn't say you should shelve the PR. I say it's an improvement. Once merged we should raise a follow up which adds the button inserter into the block when it's empty. 

That way when you click `Start empty` you see a clear next step (i.e. the button appender) rather than a disorientating (visually) empty block.

What do you think?